PusherServer/Pusher.cs

```diff
@@ -202,7 +202,21 @@ private IRestResponse ExecuteTrigger(string[] channelNames, string eventName, ob
 
             var request = CreateAuthenticatedRequest(Method.POST, "/events", null, requestBody);
 
+            System.Console.WriteLine("Method: {1}{0}Host: {2}{0}Resource: {3}{0}Parameters:{4}",

```

Comment:
It's fine for debugging the library but it's really not clean. The user might be using stdout to send CSV data into a file and that data is now interleaving.